### PR TITLE
FLOW-10395: Update GeoInformationService

### PR DESCRIPTION
**Pull request by nawrotw**
## Summary
This PR is about to align types with backend.
There are no braking changes in platform-app, so this PR can be merged without any dependencies
<br />

## Checklist
___
- [/] My code follows the style guidelines of this project
- [/] I have performed a self-review of my own code
- [/] I have commented my code, particularly in hard-to-understand areas
- [/] My changes generate no new warnings (ESLint)

<br />

## Links
___
- Jira: https://flowfact.atlassian.net/browse/FLOW-10395

### CODE REVIEW

File: src/service/GeoInformationService/GeoInformationService.Types.ts [hunk 1]
```diff
@@ -3,13 +3,12 @@ import { Captions } from '@flowfact/types';
 export namespace GeoInformationsServiceTypes {
     export interface GeoInformationValue {
         captions: {
```
Comment (rhoffmann) [inline src/service/GeoInformationService/GeoInformationService.Types.ts:5]: there is a type for captions (why are both de and en optional btw?)

File: src/service/GeoInformationService/GeoInformationService.Types.ts [hunk 2]
```diff
@@ -62,4 +67,17 @@ export namespace GeoInformationsServiceTypes {
         lat: number;
         lon: number;
     }
+
+    export interface CreateGeoInformationRequest {
+        captions: {
+            en?: string;
+            de?: string;
+        };
```
Comment (rhoffmann) [inline src/service/GeoInformationService/GeoInformationService.Types.ts:75]: same as above `Captions`

File: src/service/GeoInformationService/GeoInformationService.Types.ts (hunk 1 above)
Comment (nawrotw) [reply src/service/GeoInformationService/GeoInformationService.Types.ts:5]: captions are required but inside sufficient is to have 'en' or 'de'.
So if i make them without ? typescript will require to have both:
catptions: {
de: 'abc',
en: 'abc',
}
but for request to be valid only 1 is need either `de` or `en`

Maybe you know better way to model that?

Comment (rhoffmann) [reply src/service/GeoInformationService/GeoInformationService.Types.ts:5]: yeah thats a good question, was also thinking about that, the only thing i can think of is a union of 
`{ de: string, en?: never }  | { de?: never, en: string}`

maybe we should put that into the TranslatableText types.

Comment (nawrotw) [reply src/service/GeoInformationService/GeoInformationService.Types.ts]: I like the idea with union :-)

I just modified it a little bit and checked with TS compiler.
These are proper:
```
const. a = { de: '' };
const. a = { en: '' };
const. a = { de: '', en: '' };
```
This is invalid:
```
const. a = { };
const. a = { de: '', bla: 123 };
```

So it seems to be really good type here, to guard what we actually want :-)

Comment (rhoffmann) [reply src/service/GeoInformationService/GeoInformationService.Types.ts]: ah ok, you are right. my suggestion was either or, but not both :) i approve it 👍🏻

**Review by rhoffmann — APPROVED** [review]

**Pull request merged**